pkg/distributor/validate.go

```diff
@@ -42,12 +42,13 @@ var (
 	reasonTooFarInFuture               = globalerror.SampleTooFarInFuture.LabelValue()
 
 	// Discarded exemplars reasons.
-	reasonExemplarLabelsMissing    = globalerror.ExemplarLabelsMissing.LabelValue()
-	reasonExemplarLabelsTooLong    = globalerror.ExemplarLabelsTooLong.LabelValue()
-	reasonExemplarTimestampInvalid = globalerror.ExemplarTimestampInvalid.LabelValue()
-	reasonExemplarLabelsBlank      = "exemplar_labels_blank"
-	reasonExemplarTooOld           = "exemplar_too_old"
-	reasonExemplarTooFarInFuture   = "exemplar_too_far_in_future"
+	reasonExemplarLabelsMissing           = globalerror.ExemplarLabelsMissing.LabelValue()
+	reasonExemplarLabelsTooLong           = globalerror.ExemplarLabelsTooLong.LabelValue()
+	reasonExemplarTimestampInvalid        = globalerror.ExemplarTimestampInvalid.LabelValue()
+	reasonExemplarLabelsBlank             = "exemplar_labels_blank"
+	reasonExemplarTooOld                  = "exemplar_too_old"
+	reasonExemplarTooFarInFuture          = "exemplar_too_far_in_future"
+	reasonExamplarTooManyExamplerPerSerie = "too_many_exemplars_per_serie"
```

Comment:
"series" is both singular and plural.
```suggestion
	reasonExamplarTooManyExamplerPerSeries = "too_many_exemplars_per_series"
```